index.html

```diff
@@ -195,90 +203,98 @@ <h2 class="works-card-title">Multi-Post Stories</h2>
         </li>
       </ul>
     </section>
+
     <section class="about">
-      <div class="about-heading">
-        <h2 class="about-about-h2">About Myself</h2>
-        <p class="about-desc">
-          Hello I’m a software developer! I can help you build a product ,
-          feature or website Look through some of my work and experience! If you
-          like what you see and have a project you need coded, don’t hestiate to
-          contact me.
-        </p>
-        <h3 class="about-conn">Let's Connect</h3>
-        <ul class="about-icons">
-          <li class="about-icons-li">
-            <a href="#" class="about-icons-li-a"
-              ><img src="/assets/github.svg.svg" alt="github icon"
-            /></a>
-          </li>
-          <li class="about-icons-li">
-            <a href="#" class="about-icons-li-a"
-              ><img src="/assets/twitter.svg.svg" alt="github icon"
-            /></a>
-          </li>
-          <li class="about-icons-li">
-            <a href="#" class="about-icons-li-a"
-              ><img src="/assets/angellist.svg.svg" alt="github icon"
-            /></a>
+      <div class="about-me-description">
+        <div class="about-heading">
+          <h2 class="about-about-h2">About Myself</h2>
+          <p class="about-desc">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If you
+            like what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <h3 class="h3-conn">LET'S CONNECT</h3>
+          <ul class="about-icons">
+            <li class="about-icons-li">
+              <a href="#" class="about-icons-li-a"
+                ><img src="./assets/github.svg.svg" alt="github icon"
+              /></a>
+            </li>
+            <li class="about-icons-li">
+              <a href="#" class="about-icons-li-a"
+                ><img src="./assets/twitter.svg.svg" alt="github icon"
+              /></a>
+            </li>
+            <li class="about-icons-li">
+              <a href="#" class="about-icons-li-a"
+                ><img src="./assets/angellist.svg.svg" alt="github icon"
+              /></a>
+            </li>
+            <li class="about-icons-li">
+              <a href="#" class="about-icons-li-a"
+                ><img src="./assets/linkedin.svg.svg" alt="github icon"
+              /></a>
+            </li>
+            <li class="about-icons-li">
+              <a href="#" class="about-icons-li-a"
+                ><img src="./assets/medium.svg.svg" alt="github icon"
+              /></a>
+            </li>
+          </ul>
+           <div class="about-btn">
+              <button type="button" class="btn">Get My Resume</button>
+            </div>
```

Comment:
- [ OPTIOANL] this review is optional since the previous review did not mention it but I strongly recommend you take it into consideration. Your `get my resume` button is not designed as shown in the Figma design.

- <img width="282" alt="res" src="https://user-images.githubusercontent.com/79058364/155852374-b22a9b23-30ca-45a9-a404-65bea56b6b6c.png">

Reply:
* The issue has been resolved.